tests/datasource/test_new_datasource.py

```diff
@@ -236,31 +234,26 @@ def test_basic_pandas_datasource_v013_self_check(basic_pandas_datasource_v013):
     }
 
 
-def test_basic_spark_datasource_self_check(basic_spark_datasource):
+def test_basic_spark_datasource_self_check_spark_app_name(basic_spark_datasource):
```

Comment:
@anthonyburdi Thinking of proposing that we do not change the test name (for  now).

Reply:
Agree! But I'm not changing the test name, this is an additional test. The `test_basic_spark_datasource_self_check` still exists below.